README.md

```diff
@@ -82,17 +86,24 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
-
+<Sentence> ::= "The" <Adjectives> <Noun> <Verb> <Opt-Adverb>
+<Adjectives>::= "" | <Adjectives><Adjective>
+<Adjective>::= "lazy" | "smelly"
+<Opt-Adverb> ::= "" | <Adverb>
+<Adverd>::= "slowly" | "noisily"
+<Noun>::= "dog" | "cat"
+<Verb>::= "ate" | "ran"
 
 ## Q2.2 (5 points)
 
 Write this grammar using EBNF with common extensions
 
 ## A2.2
 
-«replace this with your answer»
-
+<Sentence> ::= "The" {<Adjective>} <Noun><Verb> ["slowly"|"noisily"]
+<Adjective>::= "lazy" | "smelly"
+<Noun>::= "dog" | "cat"
+<Verb>::= "ate" | "ran"
```

Comment:
No deduction, but why is <adjective> a non terminal, but adverb is inline?

This is really fine tuning what you do, but inconsistencies like that cause someone reading the code to stop and try and work out what the difference is.